lldb/unittests/Host/FileActionTest.cpp

```diff
@@ -6,6 +6,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include <fcntl.h>
```

Comment:
Will this work on windows?

Reply:
Good question, from the code search it seems a lot of base classes have that #include. I don't have a windows machine to test with but I doubt this will break windows.
eg. https://github.com/llvm/llvm-project/blob/main/lldb/source/Host/common/File.cpp#L15-L19

Reply:
It's also passing the Windows CI. @bulbazord I can help @kusmour validate if you want, but I personally think CI is good enough

Reply:
CI is good enough for me here. I've approved the PR.